src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -73,6 +74,7 @@ export class Connector implements IConnector {
         const defaultOptions = {
             loadCompleteRetryInterval: 250,
             maxLoadWaitTime: 30000,
+            newTabUrl: 'http://sonarwhal-empty-site.azurewebsites.net/',
```

Comment:
>It will actually be empty.sonarwhal.com once I'm back full time 😊

I'll change the url.

>@sarvaje Maybe add a bit of explanation why this URL is used, who owns it, etc. Thanks!

ok!